datahub-classify/tests/test_infotype_predictor.py

```diff
@@ -42,157 +43,22 @@
 ]
 
 
-def get_public_data(input_data_path):
-    print("===============%s=================" % input_data_path)
-    data1 = pd.read_csv(os.path.join(input_data_path, "UCI_Credit_Card.csv"))
-    data2 = pd.read_csv(os.path.join(input_data_path, "Age2_address1_credit_card3.csv"))
-    data3 = pd.read_csv(os.path.join(input_data_path, "list_of_real_usa_addresses.csv"))
-    data4 = pd.read_csv(os.path.join(input_data_path, "CardBase.csv"))
-    data5 = pd.read_csv(os.path.join(input_data_path, "Credit_Card2.csv"))
-    data6 = pd.read_csv(os.path.join(input_data_path, "catalog.csv"))
-    data7 = pd.read_csv(os.path.join(input_data_path, "iban.csv"))
-    # data8 = pd.read_csv(os.path.join(input_data_path, "USA_cars_datasets.csv"))
```

Comment:
why did we remove a bunch of csv files from the tests?

Reply:
We are still using all the files, but reading them from the directory instead of referring to each file in the python code

Reply:
It seems like a number of files were also removed from the tests directory?

Reply:
Sorry for the confusion, we have removed some of the datasets which didn't have a public license (as per discussion with Swaroop)

Reply:
ok sounds good